apps/backoffice-v2/src/domains/alerts/fetchers.ts

```diff
@@ -76,7 +76,12 @@ export const AlertsListSchema = z.array(
   ObjectWithIdSchema.extend({
     dataTimestamp: z.string().datetime(),
     updatedAt: z.string().datetime(),
-    subject: ObjectWithIdSchema.extend({ name: z.string() }),
+    subject: ObjectWithIdSchema.extend({
+      id: z.string(),
```

Comment:
`id` from `ObjectWithIdSchema` is the same validation as far as I can tell. No need for both.
```suggestion

```